eldev.el

```diff
@@ -3709,7 +3709,16 @@ mode output is restricted to just the version."
   (when parameters
     (signal 'eldev-wrong-command-usage `(t "Unexpected command parameters")))
   (let* ((package     (eldev-package-descriptor))
-         (description (when (fboundp 'package--get-description) (package--get-description package))))
+         (description (when (fboundp 'package--get-description)
+                        (condition-case nil
+                            (package--get-description package)
+                          ;; Description not found in README files and using source directories which triggered error.
+                          ;; Check for commentary in source directory file instead.
```

Comment:
The comment could be improved. Sth. like: "`package--get-description` will look at the root directory, thus not finding the project description file if source directories are used. Repeat its call to `lm-commentary` manually."